package NDnnProto;


message TMultiplicativeNormConst {
    optional double Unit = 1;
}

message TMultiplicativeNormState {
}

